cfy_manager/components/restservice/db.py

```diff
@@ -134,7 +134,7 @@ def _create_args_dict():
     }
     with open(constants.CA_CERT_PATH) as f:
         args_dict['ca_cert'] = f.read()
-    rabbitmq_cert_path = config['rabbitmq'].get('broker_ca_path')
+    rabbitmq_cert_path = config['rabbitmq'].get('ca_path')
```

Comment:
Why `rabbitmq_cert_path` and not `rabbitmq_ca_path` or `rabbitmq_ca_cert_path`?

Reply:
Because it's the cert used to access rabbitmq. It might be a CA cert, it might not; but it will be used to trust rabbit.

Reply:
Just a few lines later it is used as the `rabbitmq_ca_cert`, so maybe that should be updated as well, or update the above, its just confusing.

Reply:
Fair point. Updated.